CONTRIBUTING.md

```diff
@@ -18,6 +18,9 @@
 3) Check "Use auto-import"  
    Disable "Create separate module per source set"
 4) Run `git checkout .idea` in the Terminal to regain the codeStyles
+5) If you ever want to compile the project with IDEA itself, you have to open
+  `File > Settings > Java Compiler > Additional command line parameters` and insert  
+  `--add-exports javafx.controls/com.sun.javafx.scene.control.skin=ALL-UNNAMED`
```

Comment:
Multiple sources of turth. Pls just mention that javac/kotlinc/jvm need arguments that are specified in project.gradle. Potentially outdated information would be much worse that this.

Reply:
but you can't easily copy them from the Gradle file, and it includes Xlint unchecked which is not relevant for IDEA.

Reply:
There is no such thing as relevant for IDEA. Those are javac/kotlinc/jvm arguments and are relevant for javac/kotlinc/jvm. Developer who wants to do things without gradle should use all arguments we use in order to use the same dev. environment. I argue XLint and any argument we specify is relevant and desirable and makes my suggestion even more useful for everyone involved due to leading to 0 maintenance and always-correct result.

Plus, the dev should have a look at the gradle build regardless, there is many useful info to be found there. Shielding him from this is not going to help him in the long run.